src/s4u/s4u_actor.cpp

```
@@ -53,6 +53,15 @@ double s4u::Actor::getKillTime() {
   return simcall_process_get_kill_time(pimpl_);
 }
 
+void s4u::Actor::kill(int PID) {
+  msg_process_t process = SIMIX_process_from_PID(PID);
+  if(process != NULL) {
+    simcall_process_kill(process);
+  } else {
+    // maybe raise exeption or xbt_die ? 
```

Yes, we should throw an exception in this case.